Resources/Prototypes/Entities/Structures/Dispensers/base_structuredispensers.yml

```diff
@@ -63,6 +63,7 @@
       whitelist:
         components:
         - FitsInDispenser
+      priority: 9
```

Comment:
In which cases is this one needed? When I tested it on master the beaker was always the highest priority already?

Reply:
Sometimes the soda dispenser ejects the coconut water jug before the beaker, or the booze dispenser ejects whiskey or rum before. Sometimes everything works as it should, sometimes it doesn't, this line solves the times it doesn't.